core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -254,15 +254,15 @@ class TeamPolicyInternal<Kokkos::Cuda, Properties...>
   inline void impl_set_vector_length(size_t vector_length) {
     m_vector_length = vector_length;
   }
-  inline int scratch_size(int level, int team_size_ = -1) const {
+  inline size_t scratch_size(int level, int team_size_ = -1) const {
```

Comment:
I am tempted to get rid of the inline here and below while we are at it.

Reply:
Please undo these changes, I meant fixing the ones you are editing not all occurrences.
I am not blocking if others want to approve as is.